tests/unit/test_credentials.py

```diff
@@ -60,7 +65,9 @@ async def credentials_handler(self, request: web.Request) -> web.Response:
 
 
 @pytest.fixture
-async def instance_metadata_server() -> AsyncGenerator[InstanceMetadataServer, None]:
+async def instance_metadata_server(
+    use_v2: bool = True,
```

Comment:
that's not how fixtures work... either you need to use marks, another fixture, or just always include v2

Reply:
i was thinking about having 2 fixtures, but it feels like an overkill... so yeah let me revert that